pom.xml

```diff
@@ -36,6 +36,11 @@
   </scm>
 
   <dependencies>
+    <dependency>
+      <groupId>org.jvnet.hudson.plugins</groupId>
+      <artifactId>extended-read-permission</artifactId>
+      <version>3.2</version>
```

Comment:
This dependency makes no sense at all.

Extended Read permission is a core feature, all this does is replace setting [a system property](https://www.jenkins.io/doc/book/managing/system-properties/#hudson-security-extendedreadpermission) with the installation of a plugin.

Presence of that plugin does not equal the permission being enabled, in neither direction. This dependency should be removed.